models/db.js

```diff
@@ -33,6 +33,18 @@ db.otpModel = require('../models/otp')(sequelize, Sequelize);
 db.categoryModel = require('../models/category')(sequelize, Sequelize);
 db.productModel = require('../models/product')(sequelize, Sequelize);
 db.productImageModel = require('../models/product_image')(sequelize, Sequelize);
+db.cartModel = require('../models/cart')(sequelize, Sequelize);
```

Comment:
and always remeber when u have adding code in exist file at that time code write in last line not in random place